apps/main/src/modules/borrow/components/ApyBreakdown.tsx

```diff
@@ -0,0 +1,105 @@
+import { FlexProps, Text } from "@galacticcouncil/ui/components"
+import { getToken } from "@galacticcouncil/ui/utils"
+import { getAssetIdFromAddress } from "@galacticcouncil/utils"
+import Big from "big.js"
+import { useTranslation } from "react-i18next"
+
+import { ApyType, BorrowAssetApyData } from "@/api/borrow"
+import { AssetLogo } from "@/components/AssetLogo"
+import { DetailedApy } from "@/components/DetailedApy/DetailedApy"
+
+export type ApyBreakdownProps = FlexProps & {
+  type: ApyType
+  assetId: string
+  withFarms?: boolean
+  omnipoolFee?: string
+  apyData: BorrowAssetApyData
+}
+
+export const ApyBreakdown: React.FC<ApyBreakdownProps> = ({
+  type,
+  //withFarms,
+  omnipoolFee,
+  apyData,
+  ...props
+}) => {
+  const { t } = useTranslation()
+
+  const {
+    totalSupplyApy,
+    totalBorrowApy,
+    lpAPY,
+    underlyingAssetsApyData,
+    incentives,
+    // farms = [] @TODO: farms
+  } = apyData
+
+  const isSupply = type === "supply"
+  const baseApy = isSupply ? totalSupplyApy : totalBorrowApy
+  const apy = Big(baseApy ?? 0)
```

Comment:
also, I would consider sone kind of skeleton, if data is not fetched yet, instead of putting fallback